docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+fact, during each era validators can earn `era points` by doing different tasks on chain. The more the points, the higher the reward for a specific era. This promotes validators' activity on chain.
+To know more about era points, and how and on which basis they are distributed visit the
+[dedicated page](../maintain/maintain-guides-validator-payout.md). Distribution of the rewards is pro-rata to all stakers after the validator's commission is deducted.
+
+### Skin in the game when Staking
+
+The security of PoS networks depends on the amount of staked tokens. To successfully attack the network, a malicious actor would need to accrue a large number of tokens or would need different participants to collude and act maliciously. If there is an attack in the case of NPoS, both the validator(s) and nominators will be
+`slashed` resulting in their stake being partially or fully confiscated by the network and then deposited to the treasury. There is little interest for a rational network participant to act in a harmful way because NPoS ensures that all participants can be
+held accountable for their bad actions. In NPoS, validators are paid equal rewards regardless of the amount of stake backing them, thus avoiding large payouts to few large validators which might lead to centralization.
+
+## Being a nominator
+
+### Tasks and Responsibilities of a Nominator
+
+Since validator slots are limited, most of those who wish to stake their DOT and contribute to the
+economic security of the network will be nominators, thus here we focus on the role of nominators.
+However, it is worth mentioning that validators do most of the heavy lifting: they run the validator nodes and manage
+[session keys](https://research.web3.foundation/en/latest/polkadot/keys/index.html?highlight=session%20keys),
+produce new block candidates in [BABE](learn-consensus.md/#block-production-babe), vote and come to
+consensus in [GRANDPA](learn-consensus.md/#finality-gadget-grandpa), validate the state transition
+function of parachains, and possibly some other responsibilities regarding data availability and
+[XCM](learn-cross-consensus.md). For more information, you can take a look at the
+[validator docs](learn-validator.md) to understand what you need to do as a validator. If you want
+to become a validator you can consult [this](../maintain/maintain-guides-how-to-validate-polkadot.md) guide.
+
+Nominators have far fewer responsibilities than validators. These include selecting validators and
+monitoring their performance, keeping an eye on changing commission rates (a validator can change
+commission at any time), and general health monitoring of their validators' accounts. Thus, while not
+being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared to that of
+a validator. For more information, you can take a look at the [nominator guide](learn-nominator.md)
+to understand your responsibilities as a nominator.
```

Comment:
Here you should mention that one step more passive than nomination is joining a pool.

Reply:
Added